### PR TITLE
Added Curio as Easy

### PR DESCRIPTION
**Pull request by jausten95**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -3974,7 +3983,8 @@
         "url": "https://diasp.eu/user/edit#close_account_pane",
         "difficulty": "easy",
         "notes": "Click close my account and confirm with your password.",
-        "notes_tr": "\"Hesabımı Kapat\" tuşuna tıklayın ve şifrenizi girerek onaylayın.",
+        "notes_tr": "\"Hesabımı 
+        at\" tuşuna tıklayın ve şifrenizi girerek onaylayın.",
```
Comment (tupaschoal) [inline _data/sites.json:3987]:
```suggestion
        "notes_tr": "\"Hesabımı at\" tuşuna tıklayın ve şifrenizi girerek onaylayın.",
```

File: _data/sites.json [hunk 2]
```diff
@@ -3663,7 +3663,16 @@
             "help.ctrader.com"
         ]
     },
-
+    
+    {
+        "name": "Curio",
```
Comment (tupaschoal) [inline _data/sites.json:3668]:
```suggestion

    {
        "name": "Curio",
```

File: _data/sites.json [hunk 3]
```diff
@@ -3663,7 +3663,16 @@
...
+        "url": "https://curio.io/account/delete",
+        "difficulty": "easy",
+        "domains": [
+            "curio.io"
+        ]
+    },
+    
     {
```
Comment (tupaschoal) [inline _data/sites.json:3676]:
```suggestion
    },

    {
```

**Comment by tupaschoal** [comment]
Were the "notes_tr" changes intended?

**Comment by jausten95** [comment]
> Were the "notes_tr" changes intended?

No. I hadn't changed anything near that section of the file. Good find though

**Review by tupaschoal — APPROVED** [review]

**Pull request merged**